pkgs/development/python-modules/mtcnn/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, opencv3
+, tensorflow
+, keras
+}:
+
+
+buildPythonPackage rec {
+  pname = "mtcnn";
+  version = "0.1.1";
+  format = "pyproject";
```

Comment:
Should be `pyproject = true;` now (see https://github.com/NixOS/nixpkgs/commit/39c64e232328e09284c766d2b56f2d680e511d41).